src/common/util/game.js

```diff
@@ -321,7 +321,22 @@ function endTurn(state) {
 }
 
 export function drawCards(state, player, count) {
-  player.hand = player.hand.concat(player.deck.splice(0, count));
+  const numCardsDrawn = Math.min(count, MAX_HAND_SIZE - player.hand.length);
+  const numCardsDiscarded = count - numCardsDrawn;
+
+  if (numCardsDrawn > 0) {
+    player.hand = player.hand.concat(player.deck.splice(0, numCardsDrawn));
+  }
+
+  times(numCardsDiscarded, () => {
+    const card = player.deck[0];
+    player.deck.splice(0, 1);
+    state = logAction(state, player, `had to discard |${card.name}| due to having a full hand of ${MAX_HAND_SIZE} cards`, {
```

Comment:
What do you think of having cards highlighted in the feed, and when you hover over a card, it displays it.